src/main/java/frc/robot/RobotContainer.java

```diff
@@ -135,6 +144,9 @@ _gyro, new ModuleIOSparkMax(Constants.CAN.MODULE_FL_DRIVE, Constants.CAN.MODULE_
     private void configureButtonBindings()
     {
         _drive.setDefaultCommand(DriveCommands.joystickDrive(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), () -> -_joystick.getZ()));
+        new JoystickButton(_joystick, 4).whileTrue(DriveCommands.driveAtOrientation(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), 90.00, 1)); // TODO: test values
+        new JoystickButton(_joystick, 3).whileTrue(DriveCommands.aimAtSpeaker(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), 1)); // TODO: test values
+        new JoystickButton(_joystick, 2).whileTrue(DriveCommands.aimAtAmp(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), 1)); // TODO: test values
```

Comment:
Eventually I'd like to do something like last year for joystick buttons. Take a look at this:

https://github.com/Swartdogs/ChargedCode/blob/dcc4e5bae350d8c8237b0035b0677a475d172165/src/main/java/frc/robot/RobotContainer.java#L54-L83

This is how we were able to bind controls because of this:

https://github.com/Swartdogs/ChargedCode/blob/dcc4e5bae350d8c8237b0035b0677a475d172165/src/main/java/frc/robot/RobotContainer.java#L137-L144